tests/console/login.go

```diff
@@ -0,0 +1,205 @@
+package console
+
+import (
+	"fmt"
+	"regexp"
+	"time"
+
+	v1 "kubevirt.io/client-go/api/v1"
+
+	expect "github.com/google/goexpect"
+	"google.golang.org/grpc/codes"
+
+	"kubevirt.io/client-go/kubecli"
+	"kubevirt.io/client-go/log"
+	"kubevirt.io/kubevirt/pkg/util/net/dns"
+)
+
+func PanicOnError(err error) {
+	if err != nil {
+		panic(err)
+	}
+}
+
+// LoginToCirros call LoggedInFedoraExpecter but does not return the expecter
```

Comment:
`LoginToCirros` is still present in `tests/login.go`, these changes are duplicating it, shouldn't it be removed from there? Or maybe update the functions in `tests/console/login.go` to use the version in `tests/login.go`. Also, the version in this PR is not up to date, see https://github.com/kubevirt/kubevirt/blob/master/tests/login.go#L19-L26.